### ISSUE TITLE
Rename Twitter to X

### ISSUE THREAD

**Issue by dannyvw** [Kunstmaan/KunstmaanBundlesCMS#3305]
Maybe is changing the translation enough instead of all code.

**Comment by acrobat** [comment]
Yeah I would choose to just change the translations. But maybe for a transition periode keep something like "X (Twitter)"?

**Comment by dannyvw** [comment]
In our applications we changed it to "X", but X (Twitter) is fine by me.
